salt/modules/win_path.py

```diff
@@ -93,7 +86,7 @@ def get_path():
     ).split(";")
 
     # Trim ending backslash
-    return list(map(_normalize_dir, ret))
+    return ret
```

Comment:
Why are you removing normalizing the dirs? It looks like this helps to make comparisons with other normalized directories in the rest of the code. Also, there could be security considerations when removing normalizing the path.

Reply:
Hmm, this change was the fix. But now after running it, the previous behavior is still there. Actually, testing with both versions of the return cause the issue, but only on subsequent runs and not the first run. That's really suspicious...